Thumbprint/Targets/Thumbprint/Components/Chips/Chip.swift

```diff
@@ -69,8 +69,13 @@ public class Chip: Control {
     private var feedbackGenerator: UISelectionFeedbackGenerator?
 
     // MARK: - Public API
-    public init(adjustsFontForContentSizeCategory: Bool = true) {
-        self.pill = ChipPill(adjustsFontForContentSizeCategory: adjustsFontForContentSizeCategory)
+    public convenience init(adjustsFontForContentSizeCategory: Bool = true) {
+        let pill = ChipPill(adjustsFontForContentSizeCategory: adjustsFontForContentSizeCategory)
+        self.init(pill: pill)
+    }
+
+    public init(pill: ChipPill) {
```

Comment:
Looks like you have a build error-
```
/Users/runner/work/thumbprint-ios/thumbprint-ios/Thumbprint/Targets/Thumbprint/Components/Chips/Chip.swift:77:12: error: initializer cannot be declared public because its parameter uses a private type
    public init(pill: ChipPill) {
           ^          ~~~~~~~~
/Users/runner/work/thumbprint-ios/thumbprint-ios/Thumbprint/Targets/Thumbprint/Components/Chips/Chip.swift:203:15: note: type declared here
private class ChipPill: Pill {
              ^
```

I think you just have to update `ChipPill` to be `open` as well